src/test/java/cms/gov/madie/measure/utils/QiCoreJsonUtilTest.java

```diff
@@ -488,8 +499,107 @@ public void testGetTestCaseGroupPopulationsFromMeasureReportNoCount()
+    TestCase tc1 =
+        TestCase.builder().id("TC1").name("TC1").patientId(UUID.randomUUID()).json(json).build();
+    // String baseUrl = "https://myorg.com";
+    // ReflectionTestUtils.setField(testCaseService, "madieJsonResourcesBaseUri", baseUrl);
```

Comment:
We could remove these ?

Reply:
@RohitKandimalla Removed. Thanks so much Rohit!